bedrock/firefox/redirects.py

```diff
@@ -119,9 +119,6 @@ def firefox_mobile_faq(request, *args, **kwargs):
     redirect(r'^seamonkey-transition\.html$',
              'http://www-archive.mozilla.org/seamonkey-transition.html'),
 
-    # bug 1121082
-    redirect(r'^hello/?$', 'firefox.hello'),
```

Comment:
Any reason why these existing redirects are being removed? We should probably update their destination as opposed to remove them, to avoid serving 404's.